src/constants.py

```diff
@@ -70,6 +70,7 @@
 STATIC_EATERIES_URL = GIT_CONTENT_URL + '/DiningStack/master/DiningStack/externalEateries.json'
 STATIC_CTOWN_HOURS_URL = GIT_CONTENT_URL + '/DiningStack/master/DiningStack/externalHours.json'
 STATIC_MENUS_URL = GIT_CONTENT_URL + '/DiningStack/master/DiningStack/hardcodedMenus.json'
+STATIC_EXPANDED_ITEMS_URL = GIT_CONTENT_URL + '/DiningStack/master/DiningStack/expandedItems.json'
```

Comment:
we should make a dining stack url, looking at this one and the few above, we are repeating ourselves a lot